.github/pull_request_template.md

```diff
@@ -13,12 +17,11 @@
 - [ ] **slack_sdk.scim** (SCIM API client)
 - [ ] **slack_sdk.audit_logs** (Audit Logs API client)
 - [ ] **slack_sdk.rtm_v2** (RTM client)
-- [ ] `/docs-src` (Documents, have you run `./scripts/docs.sh`?)
-- [ ] `/docs-src-v2` (Documents, have you run `./scripts/docs-v2.sh`?)
+- [ ] `/docs-src` (Documents, have you run `./scripts/docs.sh`/`./scripts/generate_api_docs.sh`?)
```

Comment:
The `./scripts/generate_api_docs.sh` does not use docs-src files at all. Thus, this sounds a bit incorrect. Adding the step to run this script to the maintainer guide section may be better.

Reply:
I see, I've removed the reference to `./scripts/generate_api_docs.sh`, let me know what you think